app/controllers/qa/linked_data_terms_controller.rb

```diff
@@ -189,6 +189,11 @@ def jsonld?
       format == 'jsonld'
     end
 
+    def context?
+      context = params.fetch(:context, false)
+      context == 'true' ? true : false
```

Comment:
@elrayle Do you care about TRUE or True here?

Reply:
Yep... I'll downcase before comparison.  Good catch.